splot/_viz_value_by_alpha_mpl.py

```diff
@@ -114,10 +114,12 @@ def vba_choropleth(x, y, gdf, cmap='GnBu',
     
     Parameters
     ----------
-    x : array
-        Variable determined by color
-    y : array
-        Variable determining alpha value
+    x_var : string or array
+            The name of variable in gdf determined by color or an array 
```

Comment:
attention here, standard docstring formatting according to pep8 is:

x[]:[]explanation
[][][][]explanation ([]=space, only 4 spaces) getting into details here now ;)